modules/weblog/init.go

```diff
@@ -69,11 +69,33 @@ func (w *WebLog) createCustomFields() error {
 	return nil
 }
 
+func (w *WebLog) createCustomTimeFields() error {
+	if len(w.CustomTimeFields) == 0 {
+		w.Debug("skipping custom time fields creating, no custom time fields provided")
+		return nil
+	}
+
+	w.Debug("starting custom time fields creating")
+	w.customTimeFields = make(map[string][]float64)
+	for i, ctf := range w.CustomTimeFields {
+		if ctf.Name == "" {
+			return fmt.Errorf("create custom field: name not set (field %d)", i+1)
+		}
+		w.customTimeFields[ctf.Name] = ctf.Histogram
+		w.Debugf("created time field '%s', histogram '%T'", ctf.Name, ctf.Histogram)
```

Comment:
```suggestion
		w.Debugf("created time field '%s', histogram '%v'", ctf.Name, ctf.Histogram)
```

Reply:
> [ DEBUG ] web_log[nginx] init.go:85 created time field 'time1', histogram '[]float64'
[ DEBUG ] web_log[nginx] init.go:85 created time field 'time2', histogram '[]float64

We are not interested in type but in actual value.